### PR TITLE
Add an explainer for the Centralized Consent API.

### PR DESCRIPTION
**Pull request by kcheney1**
A WebKit explainer for a centralized mechanism to address data selling/sharing banners on websites.

### CODE REVIEW

File: CentralizedConsentAPI/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  a few crucial ways.
+  * GPC actively pushes the Sec-GPC value to all sites via HTTP header, and is relying on legal enforcement under CCPA/GDPR to drive adoption. By contrast, the Centralized Consent API will rely on a site’s incentive to create a better user experience and have more persistent storage for consent data. It may use legal enforcement if applicable once a site obtains a user’s preference.
+  * The Centralized Consent API avoids adding a global fingerprinting bit by making per-site and global preferences indistinguishable, and mitigates fingerprinting risks further by only revealing a user’s preference in a first-party context with user interaction.
+
+
+## Acknowledgements
+
+Many thanks for valuable feedback and advice from other members of the WebKit team, including Tess, Brent, John and Simon.
+
+[CSS Pseudo-Elements]: https://drafts.csswg.org/css-pseudo-4/
```
Comment (hober) [inline CentralizedConsentAPI/README.md:156]: It looks like all of the lines from here to the end of the document are things accidentally copied from another explainer?

File: CentralizedConsentAPI/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Centralized Consent API
+
+## Authors:
+
+- Kate Cheney
```
Comment (hober) [inline CentralizedConsentAPI/README.md:5]: Your name should be a link, either to your GitHub profile, website, email address, something like that.

**Review by hober — CHANGES_REQUESTED** [review]
Looks good! You should probably remove all the junk at the end before we merge, though.

**Comment by tomayac** [comment]
The document uses a mix of curly apostrophes (’) and straight apostrophes ('), which should probably be rectified.

**Review by hober — CHANGES_REQUESTED** [review]
Please also add a link to this explainer in the top-level README.md

File: CentralizedConsentAPI/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  practical autonomy over their data.
+
+## JavaScript API
+
+* `document.dataConsentPreference():` Returns a promise with the user's
+  personal data selling/sharing preference. Rejects the promise if the user's
+  preference is unknown.
+* `document.setDataConsentInformation():` Sets up to two bits of consent
+  information if a user has not set a preference. This will be cleared on a
+  rolling 90 day basis with user interaction.
```
Comment (dmarti) [inline CentralizedConsentAPI/README.md:63]: Without user interaction? (If a user does not interact with the site for 99 days, the consent information will be cleared?)

Comment (kcheney1) [reply CentralizedConsentAPI/README.md:63]: Good catch, this could probably even be worded more clearly. I'll change it.

**Review by hober — APPROVED** [review]

**Pull request merged**